<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRSBN</class>
 <widget class="QDialog" name="DialogRSBN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select correction beacon</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>400</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>500</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>250</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="title">
    <string>Selected RSBN beacon</string>
   </property>
   <widget class="QLabel" name="labelIWPName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>KSFO (US)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWPLatlon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Latitude: 52*25.49
Longitude: 044*03.32</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWPType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>VOR/DME 113.625</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWPMagVar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Magnetic Declination:    -13.7</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWPNote">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Source: Loaded from XNVU flightplan</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIWPName2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Schola</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>SAN FRANSISCO INTERNATIONAL AIRPORT</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listRSBN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxVORDME">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>VOR/DME correction</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Max distance (KM)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>330</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>100</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRSBN</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogRSBN</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
